streaming_providers/premiumize/utils.py

```
@@ -1,140 +1,141 @@
-            create_or_get_folder_id(pm_client, info_hash)
+        torrent_folder_data = await pm_client.get_folder_list(
+            await create_or_get_folder_id(pm_client, info_hash)
         )
     else:
-        torrent_folder_data = pm_client.get_folder_list(torrent_info["folder_id"])
-    exact_match = next(
-        (f for f in torrent_folder_data["content"] if f["name"] == filename), None
+        torrent_folder_data = await pm_client.get_folder_list(torrent_info["folder_id"])
+
+    torrent_info = {
+        "files": file_data
```

Ruff

Dictionary comprehension uses static key: `"files"`

(B035)